### ISSUE TITLE
Doesn't work with VS 2017 or later

### ISSUE THREAD

**Issue by wizofaus** [TryingToImprove/SqlSyntaxHighlighting#4]
Any planned support?

**Comment by TryingToImprove** [comment]
No, I have not touched this project for a long time.. @wizofaus Are you using it?

**Comment by wizofaus** [comment]
I found another fork that worked with 2017 so all good, thanks. Surprised it's not more popular though.

**Comment by TryingToImprove** [comment]
@wizofaus Yea, but the repo here had a lot of bugs, and often SQL-queries are being dynamically generated;

```
var sql = "SELECT * FROM table";
if (id != "") sql += "WHERE Id = '123'"
``` 

Cases like that was not really possible.


----

Maybe it will be better now, since `var sql = $"SELECT * FROM Table {id != "" ? "WHERE Id = '123'" : string.Empty}"` is possible.. Idk, but I am not going to update the repository..

Which fork did you use?

**Comment by wizofaus** [comment]
https://github.com/liangbug/SqlSyntaxHighlighting - not sure what you mean about "a lot of bugs", your examples all highlight as expected for me. The "bugs" are more that words like "from", "and", "or", "if" get highlighted in strings that aren't sql at all, but it's a very minor distraction.
